L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -736,7 +736,7 @@ namespace trklet {
     bool doMultipleMatches_{true};
 
     // if true, run a dummy fit, producing TTracks directly from output of tracklet pattern reco stage
```

Comment:
Add comment saying fakefit_=true also required if using TrackerTFP/interface/KalmanFilter.h track fit.

Reply:
done

Reply:
We'll still need fakefit_=false for displaced tracking (using old KF). For now I suggest: (1) Leaving fakefit_=false in Settings.h; (2) Add function setFakeFit(bool) to Settings.h (near L60, where similar functions, such as setDTCLinkFile() are); (3) Add cfg param "Settings=cms.bool(False)" to L1Trigger/TrackFindingTracklet/python/Tracklet_cfi.py; (4) Call setFakeFIt(bool) from L1Trigger/TrackFindingTracklet/plugins/L1FPGATrackProducer.cc , setting boolean according to value in the cfg file, copying the few-line example of setDTCLinkFile() there. (5) Set cfg Settings to True in HybridTracksNew_cfg.py